guides/user/canary/stage/index.md

```diff
@@ -0,0 +1,169 @@
+---
+layout: single
+title:  "Add a Canary stage to a pipeline"
+sidebar:
+  nav: guides
+---
+
+{% include toc %}
+
+
+If you have [enabled canary for your application](/setup/canary/) and have one
+or more [canary configs prepared](), you can now add a canary stage to your
+pipeline and configure it to perform canary analysis for your deployment,
+against the target environment.
+
+This stage type is for the canary _analysis_. The canary _deployment_ is specified
```

Comment:
I think it can be made even clearer that you are responsible for deploying your baseline & canary

Reply:
I agree. We can say that the current stage does not (yet) perform any provisioning or cleanup operations for you, and you are responsible for configuring this elsewhere in your pipeline.